src/stacks-hierarchy/Root_ConfirmationScreen.tsx

```diff
@@ -0,0 +1,70 @@
+import {RootStackScreenProps} from '@atb/stacks-hierarchy/navigation-types';
+import {View} from 'react-native';
+import {ThemeText} from '@atb/components/text';
+import {StyleSheet, useTheme} from '@atb/theme';
+import ConfirmSvg from '@atb/assets/svg/mono-icons/actions/Confirm';
+import {useEffect} from 'react';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import {InteractiveColor, StaticColor} from '@atb/theme/colors';
+
+type Props = RootStackScreenProps<'Root_ConfirmationScreen'>;
+
+const DEFAULT_DELAY_BEFORE_COMPLETED = 5000;
+const CIRCLE_SIZE = 80;
+const themeColor: StaticColor = 'background_accent_0';
+const circleColor: InteractiveColor = 'interactive_2';
+
+export const Root_ConfirmationScreen = ({
+  navigation,
+  route: {
+    params: {message, delayBeforeCompleted, nextScreen},
+  },
+}: Props) => {
+  const styles = useStyles();
+  const {theme} = useTheme();
+
+  useEffect(() => {
+    const timer = setTimeout(
+      () => navigation.navigate(nextScreen.screen, nextScreen.params),
```

Comment:
By removing the generic as mentioned here: https://github.com/AtB-AS/mittatb-app/pull/4237/files#r1461808581 the compiler complains:

![image](https://github.com/AtB-AS/mittatb-app/assets/43450882/381cd0b3-1ea5-4733-8928-965f1f61a66d)

Reply:
I leave the generic set in order to avoid this issue, lets have a look when implementing.